vhdbuilder/packer/test/run-pretest.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -euo pipefail
+
+components=$(jq .ContainerImages[] --monochrome-output --compact-output < vhdbuilder/packer/components.json)
+for component in ${components[*]}; do
+	downloadURL=$(echo ${component} | jq .downloadURL)
+	downloadURL=$(echo ${downloadURL//\*/} | jq 'sub(".com/" ; ".com/v2/") | sub(":" ; "/tags/list")' -r)
+	versionsToBeDownloaded=$(echo "${component}" | jq .versions[])
+
+	validVersions=$(curl -L https://$downloadURL | jq .tags[])
```

Comment:
nit: can you do curl -sL so that the output will be silent, this will remove the download progress part:

```
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   340  100   340    0     0   6415      0 --:--:-- --:--:-- --:--:--  6415
```